styles.css

```diff
@@ -0,0 +1,330 @@
+}
+
+main {
+  display: flex;
+  width: 100%;
+}
+
+main aside {
+  display: flex;
+  flex-direction: column;
+  width: 15%;
+  height: 90vh;
+  overflow-y: auto;
+}
+
+main aside .nav {
+  display: flex;
+  flex-direction: column;
+}
+
+main aside .nav div {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+main aside a {
+  flex-grow: 1;
+  align-content: center;
+  align-items: center;
+  text-decoration: none;
+  line-height: 0.5rem;
+  font-size: 0.7rem;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  display: inline-flex;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: bold;
+  padding: 0.7rem;
+  margin-left: 0.2rem;
+  color: black;
+  border-radius: 3px;
+}
+
+main aside a:hover {
+  background-color: rgb(216, 205, 205);
+}
+
+main aside .nav .nav7 {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: aquamarine;
+}
+
+main aside .nav .nav7 span {
+  font-size: 0.5rem;
+  font-family: Roboto, Arial, sans-serif;
+  color: rgb(78, 77, 77);
+  margin-top: 0.2rem;
+  margin-left: 0.6rem;
+  margin-bottom: 0.2rem;
+  word-spacing: -0.6px;
+}
+
+main section {
+  width: 85%;
+}
+
+main section .topics {
+  display: flex;
+  width: 100%;
+  justify-content: space-evenly;
+  flex-wrap: nowrap;
+  padding: 10px 5px 5px 5px;
+  overflow-x: hidden;
+}
+
+main section .topics a {
+  font-family: Roboto, Arial, sans-serif;
+  text-decoration: none;
+  color: black;
+  font-size: 0.7rem;
+  white-space: nowrap;
+  padding: 5px 3px;
+  background-color: rgb(216, 205, 205);
+  border-radius: 3px;
+}
+
+main section .topics a:last-child {
+  position: sticky;
+  right: 10px;
+  background-color: rgb(216, 205, 205);
+  border-radius: 50%;
+}
+
+main section .topics a:first-child {
+  position: sticky;
+  left: 10px;
+  background-color: rgb(216, 205, 205);
+  border-radius: 50%;
+}
+
+main section .videos {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+  overflow: auto;
+  height: 85vh;
+}
+
+main section .videos figure {
+  display: flex;
+  flex-direction: column;
+}
+
+main section .videos figcaption {
+  color: #0f0f0f;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 14px;
+}
+
+main section .videos p {
+  color: #606060;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+}
+
+@media screen and (max-width: 1300px) {
+  main {
+    width: 100%;
+    flex-grow: 1;
+  }
+
+  main .all {
+    width: 100%;
+  }
+
+  main aside {
+    width: 10%;
+    min-width: 100px;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  main aside nav .nav2,
+  main aside nav .nav3,
+  main aside nav .nav4,
+  main aside nav .nav5,
+  main aside nav .nav6,
+  main aside .nav .nav7 {
+    display: none;
+  }
+
+  main aside h5 {
+    display: none;
+  }
+
+  main aside .nav .nav1 {
+    margin-top: 15px;
+    justify-content: flex-start;
+    align-items: center;
+  }
```

Comment:
This couple's class has unique properties, this one's is fine, so you can refactor the code like this

```html
<tagHtml class="display-none">Some stuffs!</tagHtml>
<tagHtml class="display-none">Some stuffs!</tagHtml>
<tagHtml class="display-none">Some stuffs!</tagHtml>
<tagHtml class="display-none nav nav1">Some stuffs!</tagHtml>
```